CMakeLists.txt

```
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.19)
```

```suggestion
cmake_minimum_required(VERSION 3.14.4)
```
https://www.ros.org/reps/rep-2000.html#humble-hawksbill-may-2022-may-2027

Are you using any features above 3.14.4? If not, set it to what REP-2000 recommends.